task1.py

```diff
@@ -0,0 +1,5 @@
+eleks_employees = ['Misha', 'Dima', 'Marina']
+toshiba_employees = ['Alex', 'Den', 'Marina']
+
+toshiba_employees.extend(eleks_employees)
+print(toshiba_employees)
```

Comment:
Not fully completed. Eleks still have employees after your manipulation, but it should not.

Reply:
should i clear elex_employees list after extend toshiba?

Reply:
@bohdandovbsyh